README.md

```diff
@@ -1 +1,120 @@
-# FPGA-TrafficLight
\ No newline at end of file
+# FPGA-TrafficLight
+
+# FPGA-TrafficLight
+
+>  2022 Spring NCKU FPGA Course
+>
+>  Homework 1 
+>
+>  E24076239 E24076297 E24076750
+
+## Problem1: RGB LED
+
+### Introduction
+
+This project uses **PYNQ-Z2** to implement an RGB control circuit.
+
+#### Mode
+
+| Switch | Color  |
+| ------ | ------ |
+| 00     | White  |
+| 01     | Red    |
+| 10     | Green  |
+| 11     | Yellow |
+
+### Modules
+
+`Decoder.v` is the only module used in this project which maps the input switch to output RGB color. 
+
```

Comment:
We can simply add how we decode the signal from the input. Like: “We decoded the input signal using a `case` statement. According to the input we directly output the corresponding to the RGB LEDs”